### PR TITLE
Update to OpenXR 1.0.34

### PR DESCRIPTION
**Pull request by rasmusgo**
Here are some relatively simple changes before going all the way to OpenXR 1.1.36.

### CODE REVIEW

File: openxr/src/generated.rs [hunk 1]
```diff
@@ -3127,6 +3129,9 @@ pub mod raw {
     use sys::pfn;
     #[derive(Copy, Clone)]
     pub struct Instance {
+        pub negotiate_loader_runtime_interface: pfn::NegotiateLoaderRuntimeInterface,
+        pub negotiate_loader_api_layer_interface: pfn::NegotiateLoaderApiLayerInterface,
+        pub create_api_layer_instance: pfn::CreateApiLayerInstance,
```
Comment (Ralith) [inline openxr/src/generated.rs:3250]: I don't think these are supposed to be application-facing. Should we be exposing them?

File: generator/src/main.rs [hunk 2]
```diff
@@ -1115,6 +1115,14 @@ impl Parser {
             } else {
                 quote! { #[derive(Copy, Clone, Debug, Default, PartialEq)] }
             };
+            let aliases = self.struct_aliases.iter().map(|(alias, alias_to)| {
+                if alias_to == name {
```
Comment (Ralith) [inline generator/src/main.rs:1119]: style nit: invert condition and return early to reduce rightward drift

**Review by Ralith — CHANGES_REQUESTED** [review]

File: generator/src/main.rs (hunk 2 above)
Comment (rasmusgo) [reply generator/src/main.rs:1119]: How would that work with `quote! {}` here?

File: openxr/src/generated.rs (hunk 1 above)
Comment (rasmusgo) [reply openxr/src/generated.rs:3250]: I'm not sure how they compare to the rest of the function pointers. What are the trade-offs?

File: generator/src/main.rs (hunk 2 above)
Comment (Ralith) [reply generator/src/main.rs:1119]:
```rs
if alias_to != name {
    return quote! {};
}
quote! { ... }
```

File: openxr/src/generated.rs (hunk 1 above)
Comment (Ralith) [reply openxr/src/generated.rs:3250]: Other function pointers are intended to be exposed by the loader and called by applications. These are, at a glance, intended to be exposed by runtimes/layers and called by the loader.

**Comment by rasmusgo** [comment]
- [ ] Links to `https://www.khronos.org/registry/OpenXR/specs/1.0/html/xrspec.html` should be updated to 1.1.

File: openxr/src/generated.rs (hunk 1 above)
Comment (rasmusgo) [reply openxr/src/generated.rs]: Ok, I added a blacklist to remove these from the high level bindings.

Comment (Ralith) [reply openxr/src/generated.rs]: Is there nothing at all distinguishing them in the XML?

Comment (rasmusgo) [reply openxr/src/generated.rs]: Only a comment above that block of commands.

Comment (rasmusgo) [reply openxr/src/generated.rs]:
```xml
    <!-- SECTION: OpenXR loader command definitions -->
    <!-- openxr_loader_negotiation.h -->
    <commands>
        <command successcodes="XR_SUCCESS" errorcodes="XR_ERROR_INITIALIZATION_FAILED">
            <proto><type>XrResult</type> <name>xrNegotiateLoaderRuntimeInterface</name></proto>
            <param>const <type>XrNegotiateLoaderInfo</type>* <name>loaderInfo</name></param>
            <param><type>XrNegotiateRuntimeRequest</type>* <name>runtimeRequest</name></param>
        </command>
        <command successcodes="XR_SUCCESS" errorcodes="XR_ERROR_INITIALIZATION_FAILED">
            <proto><type>XrResult</type> <name>xrNegotiateLoaderApiLayerInterface</name></proto>
            <param>const <type>XrNegotiateLoaderInfo</type>* <name>loaderInfo</name></param>
            <param len="null-terminated">const <type>char</type>* <name>layerName</name></param>
            <param><type>XrNegotiateApiLayerRequest</type>* <name>apiLayerRequest</name></param>
        </command>
        <command successcodes="XR_SUCCESS" errorcodes="XR_ERROR_INITIALIZATION_FAILED">
            <proto><type>XrResult</type> <name>xrCreateApiLayerInstance</name></proto>
            <param>const <type>XrInstanceCreateInfo</type>* <name>info</name></param>
            <param>const <type>XrApiLayerCreateInfo</type>* <name>layerInfo</name></param>
            <param><type>XrInstance</type>* <name>instance</name></param>
        </command>
    </commands>
```

Comment (Ralith) [reply openxr/src/generated.rs]: That's annoying. Guess we'll just have to stay vigilant.

**Pull request merged**